ClientSideEncryptionToCPKN/localKeyClientSideToCustomerManagedServerSide/samplesrc/Migration.cs

```diff
@@ -0,0 +1,130 @@
+﻿using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Azure.Storage.Blobs.Specialized;
+using Azure.Storage;
+using Azure.Core;
+using Azure.Identity;
+using System;
+using System.IO;
+using System.Configuration;
+
+namespace localKeyClientSideToCustomerManagedServerSide
+{
+    class Migration
+    {        
+        private static void CSEtoSSE(
+            string connectionString,
+            string containerNameString,
+            string fileNameString,
+            string filePathString,
+            ClientSideEncryptionOptions clientSideOption,
+            string encryptionScopeNameString)
+        {
+            //Download and decrypt Client Side Encrypted blob using BlobClient with Client Side Encryption Options
+            string downloadFilePath = filePathString.Replace(".txt", "Download.txt");
+            BlobClient blobClient = new BlobClient(
+                connectionString,
+                containerNameString,
+                fileNameString).WithClientSideEncryptionOptions(clientSideOption);
+            BlobDownloadInfo download = blobClient.Download();
+
+            Console.WriteLine("\nDownloading blob to \n\t{0}\n", downloadFilePath);
+
+            using (FileStream downloadFileStream = File.OpenWrite(downloadFilePath))
+            {
+                download.Content.CopyTo(downloadFileStream);
+                downloadFileStream.Close();
+            }
+            
+            //Set Blob Client Options with the created Encryption Scope
+            BlobClientOptions blobClientOptions = new BlobClientOptions()
+            {
+                EncryptionScope = encryptionScopeNameString
+            };
+
+            //Reupload Blob with Server Side Encryption
+            blobClient = new BlobClient(
+                connectionString,
+                containerNameString,
+                fileNameString.Replace(".txt", "CMK.txt"),
+                blobClientOptions);
+            using FileStream uploadFileStream = File.OpenRead(downloadFilePath);
+            blobClient.Upload(uploadFileStream, true);
+            uploadFileStream.Close();
+        }
+
+        /*
+        * Program uploads an example client side encrypted blob, and migrates it to server side encryption using Customer Managed Keys
```

Comment:
Maybe we should add here that it's encrypting with Encryption Scope with Customer Managed Key